scania-digital-design-system/sdds
# AB#474 Feature/dropdown

**Describe pull-request**  
- Added dropdown styling
- Added dropdown interaction

**Solving issue**  
_Add which issue this pull-request solves by adding # plus the number of the issue (for example #123)_
Fixes: [AB#474](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/474)

**How to test**  
- Check Figma for styling guidelines
- Check if dropdown works in different browser
- Basic dropdown (not include different types)

- [ ] Label inside
- [ ] Label outside
- [ ] Error state
- [ ] Helper text

NOT INCLUDED in this PR
-  Multi select
-  Filter
-  Nested
